src/test/java/teammates/test/cases/action/StudentProfilePictureActionTest.java

```diff
@@ -275,34 +275,6 @@ private void testActionWithEmailAndCourseUnauthorisedInstructorOrStudentMasquera
                     + " idOfInstructor1OfCourse1 without admin permission.", uae.getMessage());
         }
 
-        gaeSimulation.loginAsAdmin("admin.user");
-        ______TS("Failure case: masqueraded instructor not from same course");
```

Comment:
I believe these cases should be added back? Just that they should use actual instructor accounts instead of an admin account to test.

Reply:
There are already tests above to check for that. I think the point of these tests are exactly the opposite of our planned solutions: allowing admin to access all pictures.

Reply:
I think the tests are for testing what happens when instructors masquerade as students or other instructors, and not for testing admin accounts?

Reply:
Yep. But this PR allows admin to view all pictures regardless of whether masquerade mode is on or not, so it will fail these tests.

Reply:
Sorry for my hasty reply, which I have deleted. I don't think these tests are for that, as the 2 tests directly above are for instructor masqueraded as instructor, and student masqueraded as student. In these 2 tests actual instructor/student accounts are used. So I think if the test writer wants to use account instructor/student account, he/she could have done so?

Reply:
I believe the admin account was used for convenience.

Reply:
I probably did not make my point very clear. Let me try again:

In these deleted test cases, the admin is masqueraded as either an instructor or a student. We are currently disagreeing on the purpose of these tests. From what I understand, you think that they test what happen when an instructor/student masquerades as another instructor/student. However, the 2 tests above these deleted tests have already tested for that, so these tests cannot test for the same thing?

Another way of looking at it: these tests are all testing masquerade mode, as seen in the name of the function `testActionWithEmailAndCourseUnauthorisedInstructorOrStudentMasquerade` as opposed to another function `testActionWithEmailAndCourseUnauthorisedInstructorOrStudent` also in this class. So for masquerade mode, you need: A masqueraded as B. Both A and B are clearly specified in the 2 tests above these deleted tests. However, in these deleted tests, only B is specified. For example, this line that we are commenting on is testing when `masqueraded instructor not from same course`. So who masqueraded as an instructor not from the same course? A student, another instructor, or admin? From the information written in the test, the only answer that seems reasonable is admin. Hence, I think these tests are explicitly testing what happen when *admin* masquerade as someone else, and not just using the admin masquerade mode for convenience like in the other file `StudentProfilePageUiTest`.

Sorry for the long comment. I hope I have made it clear this time.

Reply:
I see; my bad! 😅 Thanks for making it clear for me 👍